front/app/components/admin/SubmitWrapper/index.tsx

```diff
@@ -92,6 +92,7 @@ interface Props
   onClick?: (event: FormEvent<any>) => void;
   buttonStyle?: ButtonStyles;
   animate?: boolean;
+  testId?: string;
```

Comment:
Here also - there's probably a better way to target this - getByRole('button') perhaps?